### ISSUE TITLE
Gridview does not render inside a conditional widget

### ISSUE THREAD

**Issue by amin-nas** [EnsembleUI/ensemble#572]
See this screen:
https://studio.ensembleui.com/app/bMNDkXNXY4AaOFa3e4aG/screen/mULLCprVVZe8jOOwLqT5

steps:
1. screen loads a grid view with mocked data
2. enter "manchester" in the search box and tab out to trigger the api call
3. when the API response comes, the grid view does not render properly

**Comment by vusters** [comment]
Looks like Conditional rendered the GridView child properly the first time. If the condition changes, somehow the GridView doesn't get rendered. Column works however.

**Comment by hemish11** [comment]
Upon running the same EDL in the local emulator, here's the error I found: `JS Parsing Error: Exception Occurred while running javascript code: Line: 1 Message: i.team is undefined. Check your syntax. Detailed Error: Code: i.team.id`.

The actual issue is not with the grid view as in this image I attached there are green boxes showing that grid view is there, it that i.team is undefined and we are trying to get logo, code, and so on.

<img src="https://github.com/EnsembleUI/ensemble/assets/46677360/8dfa4bea-79dc-43f5-9cea-278839039b0f" width="400">
